openapi-meta/src/main/java/com/networknt/openapi/DefaultInjectableSpecValidator.java

```diff
@@ -0,0 +1,23 @@
+package com.networknt.openapi;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+public class DefaultInjectableSpecValidator implements InjectableSpecValidator{
+    public static String PATHS = "paths";
+    Logger logger = LoggerFactory.getLogger(DefaultInjectableSpecValidator.class);
+    @Override
+    public boolean isValid(Map<String, Object> openapi, Map<String, Object> inject) {
+        if (inject.get(PATHS) instanceof Map && openapi.get(PATHS) instanceof Map) {
+            for (Map.Entry<String, Object> path : ((Map<String, Object>) inject.get(PATHS)).entrySet()) {
+                if (((Map<String, Object>)openapi.get(PATHS)).containsKey(path.getKey())) {
```

Comment:
shouldnt we compare both path and http method, to decide whether there is a conflict or not?

Reply:
As we have very specific admin endpoint names, only compare the path should be sufficient enough. Adding the HTTP method is just nice to have I think.